tests/test_crawlers.py

```diff
@@ -390,6 +415,30 @@ def test_intersects_time_range_no_upper_limit(self):
         # no upper limit and no start_time, without intersection
         self.assertFalse(crawler._intersects_time_range(None, datetime(2019, 2, 19)))
 
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._http_get")
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._get_links")
+    def test_correct_navigation_to_download_page_for_OSISAF(self, mock_get_link, mock_http_get):
+        """Test the functionality of "get_download_url" method for OpenDAP crawler of OSISAF project """
+        mock_get_link.return_value = [
+            '/thredds/dodsC/osisaf/met.no/ice/amsr2_conc/2019/11/ice_conc_nh_polstere-100_amsr2_201911301200.nc.html']
+        expected_urls = ['https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/amsr2_conc/2019/11/catalog.html?dataset=osisaf/met.no/ice/amsr2_conc/2019/11/ice_conc_nh_polstere-100_amsr2_201911301200.nc', ]
```

Comment:
`get_download_url()` takes a string as argument, not a list.
This test only works because the `mock_get_link` makes that the value of `expected_url` is never used.
This variable should be a simple string.

Reply:
Done in the next commit